projects/norway/pelias.json

```diff
@@ -0,0 +1,66 @@
+{
+  "logger": {
+    "level": "debug",
+    "timestamp": false
+  },
+  "esclient": {
+    "apiVersion": "7.5",
+    "hosts": [
+      { "host": "elasticsearch" }
+    ]
+  },
+  "elasticsearch": {
+    "settings": {
+      "index": {
+        "refresh_interval": "10s",
+        "number_of_replicas": "0",
+        "number_of_shards": "1"
+      }
+    }
+  },
+  "api": {
+    "services": {
+      "pip": { "url": "http://pip:4200" },
+      "libpostal": { "url": "http://libpostal:4400" },
+      "placeholder": { "url": "http://placeholder:4100" },
+      "interpolation": { "url": "http://interpolation:4300" }
+    },
+    "defaultParameters": {
+      "focus.point.lat": 60.47, 
+      "focus.point.lon": 6.83
+    }
```

Comment:
This section isn't strictly required.

It applies default `focus.point.lat` and `focus.point.lon` values when those are omitted in the HTTP request.

When config value is set, Pelias will favour places nearer to this point than elsewhere in Norway, returning them higher in the results list.

If you find that's preferable for Norway then I'm :+1: with it, otherwise we might consider removing it so that results father away (eg. Oslo) don't appear to get a lower score.

Reply:
Ah thanks! I did not know. I was not the intention, so I've removed it now. Thanks!